test/TesterAzureUtils/Streaming/ProgrammaticSubscribeTestAQStreamProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Orleans.Providers.Streams.AzureQueue;
+using Orleans.Runtime.Configuration;
+using Orleans.Streams;
+using Orleans.TestingHost;
+using Tester.StreamingTests;
+using TestExtensions;
+using UnitTests.StreamingTests;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Tester.AzureUtils.Streaming
+{
+    [TestCategory("BVT"), TestCategory("Streaming"), TestCategory("Functional"), TestCategory("AQStreaming")]
+    public class ProgrammaticSubscribeTestAQStreamProvider : ProgrammaticSubcribeTestsRunner, IClassFixture<ProgrammaticSubscribeTestAQStreamProvider.Fixture>
```

Comment:
not terribly important, but we tend to prepend the specialization to the runner name.  For instance EHClientStreamTests, AQClientStreamTests, SMSClientStreamTests, and so on.